src/styles/components/_404.scss

```diff
@@ -0,0 +1,69 @@
+.error-page {
+  background-image: url("../../../public/images/background_image.png");
+  background-color: var(--background-color);
+  background-size: cover;
+  background-position: center;
+  padding: 0 1.25rem 1.25rem; // top: 0; left/bottom/right: 20px/16 (should mirror header padding)
+
+  @media (prefers-color-scheme: dark) {
+    background-image: url("../../../public/images/background_image_dark.svg");
+  }
+
+  @media (min-width: $breakpoint-scale-up) {
+    padding-bottom: 5rem; // this should mirror the header padding
+  }
+
+  &__header {
+    padding: 0;
+
+    @media (min-width: $breakpoint-scale-up) {
+      padding-top: 5rem;
+    }
+  }
+
+  &__content {
+    padding: 0 1.25rem;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
```

Comment:
More personal preference stuff, sorry! What do you think about using align-items: center? A lot of the 404 pages I've seen (google, twitter, instagram, reddit...) center align their content.

Reply:
The only reason I didn't do that here with the text is because it didn't match what we were doing on other pages. We have nothing that's all centered, but I can definitely bring it up in standup. I did change Max to be centered.

https://github.com/sparkbox/accessible-components/pull/136/commits/31748275318293445a0b25415ae18a7178b9c85d